beacon-chain/verification/data_column.go

```diff
@@ -69,26 +68,32 @@ var (
 	ErrColumnIndexInvalid = errors.New("incorrect column sidecar index")
 )
 
-type RODataColumnVerifier struct {
+type RODataColumnsVerifier struct {
```

Comment:
Why is this being changed ? We already have a batch verifier for the cases where we need to modify multiple columns at once